src/main/java/frc/robot/subsystems/LiftMechanism.java

```diff
@@ -19,6 +19,20 @@
     public double initEncoderPosition; 
     public JoystickButton winchStop; 
 
+    public enum LiftHeights {
+        DOWN(-30), LOW(-108.76), MEDIUM(-229.581146), HIGH(-533.91);
```

Comment:
What units are these in? And why are there so many decimal points?

Reply:
These are in encoder ticks. Not sure about the amount of decimals.